### PR TITLE
Add Eggdrop to Bots

### PR DESCRIPTION
**Pull request by vanosg**

### CODE REVIEW

File: _data/sw_clients.yml [hunk 1]
```diff
@@ -924,6 +924,26 @@
           - plain
           - external
           - scram-sha-256
+    - name: Eggdrop
+      link: https://github.com/eggheads/eggdrop
```
Comment (DarthGandalf) [inline _data/sw_clients.yml:928]: Can you provide any reference link in source?

Comment (vanosg) [reply _data/sw_clients.yml:928]: Sure, sorry, I wasn't sure of the intent of that line- what is this intended to reference? If code, it's kinda smashed all throughout. Maybe a reference to the config file?

Comment (ProgVal) [reply _data/sw_clients.yml:928]: You can link to the list of capabilities it requests, and add an extra link for the few extensions with no capabilities

**Comment by DarthGandalf** [comment]
Thanks!

**Pull request merged**